R/dpinput_sync.R

```diff
@@ -258,15 +215,28 @@ sync_iterate <- function(input_map, inputboard_alias, skip_sync, rewrite_ok = F,
         sync_alrt <- cli::cli_alert_warning
       }
 
+      get_remote_pin_version <- pins::pin_versions(
+        name = input_i$metadata$name,
+        board = board_object
+      )$hash
+
+      input_i$metadata$description <- to_description(input_i = input_i)
+
+      if (length(get_remote_pin_version) > 1) {
+        latest_pin_version  <- sort(get_remote_pin_version)[length(get_remote_pin_version)]
```

Comment:
This looks like it's sorting by hashes, so I'm not sure that this is guaranteed to get the latest version of the pin. Recommend sorting by pin version instead, since that will contain datetime info and can use to get the hash corresponding to the most recent pin.